shopfloor/models/stock_move.py

```diff
@@ -39,8 +39,8 @@ def split_other_move_lines(self, move_lines, intersection=False):
                 )
             split_move_vals = self._split(qty_to_split)
             split_move = self.create(split_move_vals)
-            split_move._action_confirm(merge=False)
             split_move.move_line_ids = to_move
+            split_move._action_confirm(merge=False)
```

Comment:
this is not a a fix in `shopfloor_reception`.  Can you split the commit pls?
Also, wondering: the change seems to make sense but... we didn't have any issue before?

Reply:
This is because the supplier location bypass the reservation and `action_assign` is called here https://github.com/odoo/odoo/blob/14.0/addons/stock/models/stock_move.py#L1177
which causes also the creation of a new move line here
https://github.com/odoo/odoo/blob/14.0/addons/stock/models/stock_move.py#L1324

Reply:
This issue is happening only in case of a reception

Reply:
I have done the split in 2 commits